src/Libraries/RevitNodes/Elements/PathOfTravel.cs

```diff
@@ -54,11 +54,11 @@ private PathOfTravel(RvtAnalysis.PathOfTravel rvtPathOfTravel)
       #region Public constructors
 
       /// <summary>
-      /// Calculates the longest Path of Travel of all shortest paths from rooms in the floor plan to given exit points.
+      /// Calculates the longest PathofTravel(s) of all shortest paths from rooms in the floor plan to the specified exit points.
```

Comment:
All changes of "Path of Travel" to "PathofTravel": if the element class name is meant then it should be "PathOfTravel" (capital O).

Reply:
Changes committed. Thanks for your review.